src/support.cpp

```diff
@@ -1272,17 +1272,19 @@ void AreaSupport::moveUpFromModel(const SliceDataStorage& storage,
  */
 std::pair<Polygons, Polygons> AreaSupport::computeBasicAndFullOverhang(const SliceDataStorage& storage, const SliceMeshStorage& mesh, const unsigned int layer_idx)
 {
-    Polygons supportLayer_supportee = mesh.layers[layer_idx].getOutlines();
+    Polygons outlines = mesh.layers[layer_idx].getOutlines();
     constexpr bool no_support = false;
     constexpr bool no_prime_tower = false;
-    Polygons supportLayer_supporter = storage.getLayerOutlines(layer_idx - 1, no_support, no_prime_tower);
+    Polygons outlines_below = storage.getLayerOutlines(layer_idx - 1, no_support, no_prime_tower);
+    constexpr int layers_above = 3; // this is now a random value that seems to work fine. Do we need to expose this in the front end?
+    Polygons outlines_above = layer_idx + layers_above < mesh.layers.size() ? storage.getLayerOutlines(layer_idx + layers_above, no_support, no_prime_tower) : Polygons();
```

Comment:
I like this pattern a bit more; line becomes a bit too long otherwise
```suggestion
    Polygons outlines_above;
    if (layer_idx + layers_above < mesh.layers.size())
    {
        storage.getLayerOutlines(layer_idx + layers_above, no_support, no_prime_tower);
    }
    else
    {
        Polygons();
    }
```